src/content/en/2020/resource-hints.md

```diff
@@ -144,8 +147,9 @@ Resource hints are most effective when they're used selectively ("when everythin
 
 However, this hasn't stopped some misuse of the `preload` hint, since in one instance we discovered a page dynamically adding the hint and causing an infinite loop that created over 20k new preloads.
 
+{# TODO(editors): Are emojis okay? And should there be punctuation? #}
```

Comment:
@rviscomi

Reply:
I'd prefer to remove it unless @Zizzamia feels strongly

Reply:
I removed it for now.